src/bin/deku_node.ml

```diff
@@ -158,6 +158,23 @@ let handle_ticket_balance =
       let amount = Flows.request_ticket_balance state ~ticket ~address in
       Ok { amount })
 
+(* POST /get-storage *)
```

Comment:
This additional endpoint seems unrelated to the goal of the PR. I also don't think we want to expose Core_deku state via the Deku node since this conflicts with the architecture design of Deku p (potentially). See #535.